helm/vertical-pod-autoscaler-app/values.yaml

```diff
@@ -101,7 +109,7 @@ updater:
 
 admissionController:
   # admissionController.enabled -- If true, will install the admission-controller component of vpa
-  enabled: false
+  enabled: true
```

Comment:
We enable admission controller by default, we can remove the setting in config repo afterwards

Reply:
PR https://github.com/giantswarm/config/pull/771